css/properties/-webkit-line-clamp.json

```
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-box-reflect": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-webkit-line-clamp",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "17"
+            },
+            "edge_mobile": {
+              "version_added": null
```

This would be false since the last version of Edge for Windows Mobile would be 15.  

```suggestion
              "version_added": false
```